sdk/CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Renamed `AccountHandle` to `Account`, `Account` to `AccountDetails` and `AccountDto` to `AccountDetailsDto`;
 - `AddressWrapper` renamed to `Bech32Address` and moved to `types`;
 - `Address::try_from_bech32_with_hrp` address and HRP return have been reversed;
+- Remove `AddressGenerationOptions` in favor of `GenerateAddressOptions`, which now contains the `internal` flag.
```

Comment:
Can you add a `### Removed` (after `### Changed`) section and put it there please